app/.eslintrc.json

```diff
@@ -17,7 +17,8 @@
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "react/prop-types": "off",
-    "no-warning-comments": "warn"
+    "no-warning-comments": "warn",
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false}]
```

Comment:
`devDependencies` should be true. If we are importing a dependency that is only listed in dev dependencies, we want to get a warning. Otherwise the app will work during development but not when it's built.

Reply:
Done